src/main/java/seedu/chatteroo/commands/DeleteCommand.java

```diff
@@ -0,0 +1,30 @@
+package seedu.chatteroo.commands;
+
+import seedu.chatteroo.tasks.TaskList;
+import seedu.chatteroo.ui.Ui;
+import seedu.chatteroo.storage.Storage;
+import seedu.chatteroo.tasks.Task;
+
+/**
+ * Deletes the specified task from the list of tasks.
+ */
+public class DeleteCommand extends Command {
+    private int taskNum;
+
+    /**
+     * Constructor for the DeleteCommand class.
+     * @param taskNum The task number.
+     */
+    public DeleteCommand(int taskNum) {
+        super();
+        this.taskNum = taskNum;
+    }
+
+    @Override
+    public String execute(TaskList tasks, Ui ui, Storage storage) {
```

Comment:
perhaps you can inherit the java doc and elaborate more on how you are exceuting delete?